spec/controllers/application_controller_spec.rb

```diff
@@ -39,7 +39,7 @@
           expect( controller.send(:current_user) ).to eql(user)
         end
         it "should not find the user_id" do
-          expect(token).to receive('data').and_return({})
+          expect(token).to receive('data').and_return([{}])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.